Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/ParticleSmearing.cpp

```diff
@@ -43,10 +43,10 @@ ActsExamples::ProcessCode ActsExamples::ParticleSmearing::execute(
   std::normal_distribution<double> stdNormal(0.0, 1.0);
 
   for (const auto& particle : particles) {
+    const auto phi = vh::phi(particle.unitDirection());
```

Comment:
I really would keep `VectorHelper` and not `vh`, it makes it clearer where it comes from.

Reply:
Fixed.